Feature/user-address

habanera03 opened the pull request:
# 요약
### Issues : [#12](https://github.com/f-lab-edu/planB/issues/12)
1\. 배달주소 조회 구현
2\. 배달주소 삭제 구현
3\. 배달주소 수정 구현

# 리뷰 가이드
## 공통

- 존재하는 회원 및 주소 여부 확인

Resolves: #12

hagu03 committed:
feat: 배달주소 관리

1. 배달주소 조회 구현
2. 배달주소 삭제 구현
3. 배달주소 수정 구현

Resolves: #12

f-lab-evan commented on src/main/java/com/flab/planb/controller/AddressController.java:
```diff
@@ -37,22 +42,86 @@
 
+    @GetMapping("/{memberId}")
+    public ResponseEntity<?> getAddressList(@PathVariable long memberId) {
+        log.debug("memberId : {}", memberId);
```
요청시 해당 로그를 메소드 마다 각각 다 찍고 있습니다. 
아래 방법을 찾아보고 어떻게 적용하면 될지 고민해보시길 바랍니다.

1. reverse proxy를 이용한 로깅 방법 
2. tomcat의 accesslog 로깅 방법
3. AOP나 전처리를  이용한 로깅 방법

hagu03 committed:
[Feature/user-address] 리뷰 수정사항 반영

1. AOP를 이용한 로깅

[Feature/user-address] 수정사항 반영

1. AddressRequest builder validation 추가
